external/error_report/Kconfig

```diff
@@ -0,0 +1,65 @@
+#
+# For a description of the syntax of this configuration file,
+# see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt
+#
+
+config ERROR_REPORT
+	bool "Error Report Module"
+	default n
+	---help---
+		enable Error Reporting
+
+if ERROR_REPORT
+
+config ERROR_REPORT_NENTRIES
+	int "Number of Error Records"
+	default 6
+	---help---
+		Maximum number of error records to store in memory
+
+config DEBUG_NW_ERR_REPORT_ERROR
```

Comment:
As we talked, this is not specified to nw. How about remove ```NW_```?

Reply:
Yes, you are right, I have modified the names.